lib/stats/DefaultStatsPresetPlugin.js

```diff
@@ -201,7 +201,7 @@ class DefaultStatsPresetPlugin {
 	apply(compiler) {
 		compiler.hooks.compilation.tap("DefaultStatsPresetPlugin", compilation => {
 			compilation.hooks.statsPreset
-				.for(false)
+				.for("false")
```

Comment:
https://github.com/webpack/tapable/pull/88

Reply:
I missed this one. I'll revert revert this when the new typings will be published

Reply:
published